workflows/database_upsert.go

```diff
@@ -86,7 +86,9 @@ func (workflow *databaseWorkflow) databaseDeployer(service *common.Service, stac
 		}
 		stackParams["DatabaseMasterPassword"] = dbPass
 
-		err = stackUpserter.UpsertStack(dbStackName, template, stackParams, buildDatabaseTags(workflow.serviceName, environmentName, common.StackTypeDatabase, workflow.codeRevision, workflow.repoName))
+		tags, err := concatTagMaps(service.Database.Tags, buildDatabaseTags(workflow.serviceName, environmentName, common.StackTypeDatabase, workflow.codeRevision, workflow.repoName), DatabaseTags)
```

Comment:
Just pass a struct and the `service.Database.Tags`.   Remove the `buildDatabaseTags` and just create a struct and pass reference.